.github/prepare_assets.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+pip install .
+fhirutil validate --publisher_opts="-tx n/a" --clear_output site_root/ig.ini
```

Comment:
I'm assuming this script runs from the top level git repo directory not from `.github` directory?

Reply:
I believe so